sw/device/lib/crypto/include/datatypes.h

```diff
@@ -57,261 +57,273 @@ typedef status_t crypto_status_t;
  * without changing all error codes. Remove the seed (-s argument) to generate
  * completely new 11-bit values.
  */
-typedef enum crypto_status_value {
+typedef enum otcrypto_status_value {
   // Status is OK; no errors.
-  kCryptoStatusOK = (int32_t)0x739,
+  kOtcryptoStatusOk = (int32_t)0x739,
   // Invalid input arguments; wrong length or invalid type.
-  kCryptoStatusBadArgs = (int32_t)0x8000fea0 | kInvalidArgument,
+  kOtcryptoStatusBadArgs = (int32_t)0x8000fea0 | kInvalidArgument,
   // Error after which it is OK to retry (e.g. timeout).
-  kCryptoStatusInternalError = (int32_t)0x80005340 | kAborted,
+  kOtcryptoStatusInternalError = (int32_t)0x80005340 | kAborted,
   // Error after which it is not OK to retry (e.g. integrity check).
   kCryptoStatusFatalError = (int32_t)0x80006d80 | kFailedPrecondition,
   // An asynchronous operation is still in progress.
-  kCryptoStatusAsyncIncomplete = (int32_t)0x8000ea40 | kUnavailable,
+  kOtcryptoStatusAsyncIncomplete = (int32_t)0x8000ea40 | kUnavailable,
   // TODO: remove all instances of this error before release; it is to track
   // implementations that are not yet complete.
-  kCryptoStatusNotImplemented = (int32_t)0x80008d20 | kUnimplemented,
-} crypto_status_value_t;
+  kOtcryptoStatusNotImplemented = (int32_t)0x80008d20 | kUnimplemented,
+} otcrypto_status_value_t;
```

Comment:
The prefix for this **otcrypto_status_value_t** enum's constants needs to be updated to kOtcryptoStatusValue(xyz)
(The script did its job; The incorrect naming resulted from an existing issue)